bin/test_travis.sh

```diff
@@ -11,6 +11,7 @@ python setup.py install --symengine-dir=$our_install_dir
 # Test python wrappers
 if [[ "${WITH_SAGE}" != "yes" ]]; then
     nosetests -v
+    py.test -v symengine/tests/test_*.py  # pytest gets confused if we install first
```

Comment:
let's remove nosetests, and just use py.test, as we agreed.

I think you can use `$PYTHON_SOURCE_DIR` as below for the path?